README.md

```diff
@@ -239,51 +255,9 @@ if test_track_visitor.ab :dark_deployed_feature, context: 'signup'
 end
 ```
 
-### Varying app behavior in an offline context
-
-The `OfflineSession` class can be used to load a test track visitor when there is no access to browser cookies. It is perfect for use in a process being run from either a job queue or a scheduler. The visitor object that is yielded to the block is the same as the visitor in a controller context; it has both the `vary` and `ab` methods.
-
-An `OfflineSession` can be established in one of two ways:
-
-1. with an `identifier_type`:
-```ruby
-OfflineSession.with_visitor_for(:myapp_user_id, 1234) do |test_track_visitor|
```

Comment:
I decided that we should privatize this OfflineSession stuff and lean into the Identity concept. It is much easier to work with and allows us to manage the session better when we get to that part. So I ripped the docs out of the README.